ambry-store/src/main/java/com/github/ambry/store/StorageManager.java

```diff
@@ -534,6 +540,11 @@ public boolean addBlobStore(ReplicaId replica) {
     return true;
   }
 
+  @Override
+  public boolean addFileStore(ReplicaId replicaId) {
+    return false;
```

Comment:
We should add TODOs here if pending implementations are there.

Reply:
added